src/main/java/net/minecraftforge/common/ForgeHooks.java

```diff
@@ -1012,6 +1012,22 @@ public static void onEmptyLeftClick(EntityPlayer player, ItemStack stack)
         MinecraftForge.EVENT_BUS.post(new PlayerInteractEvent.LeftClickEmpty(player, stack));
     }
 
+    public static net.minecraftforge.event.entity.player.CriticalHitEvent getCriticalHit(EntityPlayer player, Entity target, boolean vanillaCritical, float vanillaDefault)
```

Comment:
in theory, we whould move vanilla stuff to forge if we put the 1.5 in a forge class, thats why it is still in the EntityPlayer